bolt/lib/Passes/BinaryPasses.cpp

```diff
@@ -223,6 +223,22 @@ static cl::opt<unsigned> TopCalledLimit(
              "functions section"),
     cl::init(100), cl::Hidden, cl::cat(BoltCategory));
 
+// Profile density options, synced with llvm-profgen/ProfileGenerator.cpp
+static cl::opt<bool> ShowDensity("show-density", cl::init(false),
```

Comment:
In llvm-profgen, this flag is only used to print the density number(the last `outs()`), it by default enables the density calculation so that if the density is low, we emit a warning for this. But here it's a global switch, is this intentional?

Reply:
I see. We don't report density by default but I'm not against it considering it's pretty cheap to calculate.

Reply:
Let's keep it disabled by default until we find good threshold value.

Reply:
Reenabled by default with threshold=50